docs/user-documentation/2xfor1x.md

```diff
@@ -60,7 +60,7 @@ In Fedora 4:
 * Ingest occurs asynchronously soon after. 
 
 ###Collections
-Because objects in Fedora 3 were stored in a flat graph structure instead of a hierarchy, what were presented as collection in Islandora 7.x-1.x were actually objects on the same level as their child objects, with the 'container' or 'folder' aspect of them being a fiction for display created by the relationships between the objects. In Fedora 4, resources do have a true hierarchical structure and must have a `fedora:hasParent` relationship to know where they belong in a given repository. Indeed, to migrate objects over from Fedora 3 to Fedora 4, parents must arrive before their children. In its current incarnation, all objects in a 7.x-2.x repository have the Fedora root as a parent and relationships are managed via the ```pcdm:hasMember``` predicate. 
+Because objects in Fedora 3 were stored in a flat graph structure instead of a hierarchy, what were presented as collections in Islandora 7.x-1.x were actually objects on the same level as their child objects, with the 'container' or 'folder' aspect of them being a fiction for display created by the relationships between the objects. In Fedora 4, resources do have a true hierarchical structure and must have a `fedora:hasParent` relationship to know where they belong in a given repository. Indeed, to migrate objects over from Fedora 3 to Fedora 4, parents must arrive before their children. In its current incarnation, all objects in a 7.x-2.x repository have the Fedora root as a parent and relationships are managed via the ```pcdm:hasMember``` predicate. 
```

Comment:
Entirely possible this is wrong wrong wrong (or just wildly out of date?). I wrote it up without a whole lot of expertise in the specifics, because we needed general intro document. Suggested changes to update/make it more accurate are very welcome 😄